examples/workflows/map_reduce.py

```diff
@@ -0,0 +1,75 @@
+"""
+This is a map reduce example from the upstream Argo Workflows repository. This is not part of the upstream examples
+folder for workflows because the upstream example is not formatted properly
```

Comment:
Adds the markdown link to render in the docs

```suggestion
folder for workflows because the upstream example is not formatted properly.

See the upstream example [here](https://github.com/argoproj/argo-workflows/blob/master/examples/map-reduce.yaml)
```

Reply:
Added in #649